src/Files.Uwp/Views/MainPage.xaml.cs

```diff
@@ -139,6 +160,48 @@ private void HorizontalMultitaskingControl_Loaded(object sender, RoutedEventArgs
             }
         }
 
+        private void SetRectDragRegion()
+        {
+            if (!AppWindowTitleBar.IsCustomizationSupported())
+                return;
+
+            const bool WORKAROUND = true;
+            if (WORKAROUND)
+            {
+                const short LeftButton = 0x01;
+
+                if ((NativeWinApiHelper.GetKeyState(LeftButton) & 0xFF00) == 0xFF00)
+                    return;                    
+
+                App.Window.AppWindow.TitleBar.ResetToDefault();
+                App.Window.AppWindow.TitleBar.ExtendsContentIntoTitleBar = true;
+
+                // Set window buttons background to transparent
+                App.Window.AppWindow.TitleBar.ButtonBackgroundColor = Colors.Transparent;
+                App.Window.AppWindow.TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
+            }
+
+            const uint MDT_Effective_DPI = 0;
+
+            var displayArea = DisplayArea.GetFromWindowId(App.Window.AppWindow.Id, DisplayAreaFallback.Primary);
+            var hMonitor = Win32Interop.GetMonitorFromDisplayId(displayArea.DisplayId);
+            var hr = NativeWinApiHelper.GetDpiForMonitor(hMonitor, MDT_Effective_DPI, out var dpiX, out _);
+            if (hr != 0)
+                return;
+            
+            var scalePercent = (uint)(((long)dpiX * 100 + (96 >> 1)) / 96); // wtf
+            var scaleAdjustment = scalePercent / 100.0;
```

Comment:
You should be able to simplify this by replacing all the DPI code with setting `scaleAdjustment` to `XamlRoot.RasterizationScale`.

Reply:
@d2dyno1

Reply:
This should be addressed with #9945